doc/examples/config_kube_bearer_token.rst

```diff
@@ -3,7 +3,7 @@
    $ cat /etc/commissaire/commissaire.conf
    {
      ...
-     "register-store-handler": [
+     "storage-handlers": [
        "name": "commissaire.store.kubestorehandler",
```

Comment:
Since we ditched the kube handler, change the name to `"commissaire.storage.etcd"`.